samples/spring-boot-external-config/AppConfig.pkl

```diff
@@ -0,0 +1,14 @@
+// this module name determines the package and
+// class name of the generated Java config class
+module samples.boot.AppConfig
+
+server: Server
+
+class Server {
+  endpoints: Listing<Endpoint>
+}
+
+class Endpoint {
+  name: String
+  port: UInt16
+}
```

Comment:
Can you move this file also to `config`?

It seems somewhat strange that `AppConfig.pkl` is at the root of the project, when there's also a `config/` directory

Reply:
Actually, I went ahead and fixed these all up for you